app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -116,6 +118,31 @@ const MeetingEditor = () => {
     })
   );
 
+  const updateReportValue = useCallback(
+    (newReportValue) => {
+      const debouncedFunction = debounce(() => {
+        const storedReport = localStorage.getItem(`meeting-${meetingId}-temp`);
+        if (!storedReport || storedReport.length < newReportValue.length) {
+          setReportValue(newReportValue);
+        }
+      }, 4500);
+
+      debouncedFunction();
```

Comment:
Any reason this is declared as a function and not called directly?

`debounce(() => {...}, 4500);` or `debounce(() => {...}, 4500)();` depending on which works